package.json

```diff
@@ -112,15 +114,17 @@
   },
   "scripts": {
     "dev": "webpack --progress --colors --mode development --config scripts/webpack/webpack.dev.js",
-    "start": "webpack-dev-server --progress --colors --mode development --config scripts/webpack/webpack.hot.js",
+    "start": "node ./scripts/cli/start.js",
     "watch": "webpack --progress --colors --watch --mode development --config scripts/webpack/webpack.dev.js",
     "build": "grunt build",
     "test": "grunt test",
     "tslint": "tslint -c tslint.json --project tsconfig.json",
     "typecheck": "tsc --noEmit",
     "jest": "jest --notify --watch",
     "api-tests": "jest --notify --watch --config=tests/api/jest.js",
-    "storybook": "cd packages/grafana-ui && yarn storybook"
+    "precommit": "grunt precommit",
```

Comment:
this pre-commit hook can't be here or needs to be the same as the one in husky: hooks otherwise husky will use this (it's the old way for husky)

Reply:
OK, just added it for testing issue with TS